e2etest/scenario_os_helpers_for_windows.go

```diff
@@ -28,9 +28,11 @@ import (
 	"strings"
 	"syscall"
 	"time"
+	"unsafe"
 
 	"github.com/Azure/azure-storage-azcopy/v10/common"
 	"github.com/Azure/azure-storage-azcopy/v10/ste"
+	"github.com/hillu/go-ntdll"
```

Comment:
Please update the license file for this package. 
Also, Is this a known repository?

Reply:
We already use this package-- It was required for fixing up folder times.

Reply:
(Line 9371 on NOTICE.txt)